bigbluebutton-html5/imports/ui/components/video-provider/video-menu/component.jsx

```diff
@@ -50,10 +50,12 @@ const JoinVideoOptions = ({
       ));
   return (
     <Dropdown
+      className={styles.container}
       autoFocus
     >
       <DropdownTrigger tabIndex={0}>
         <Button
+          className={styles.button}
```

Comment:
have any issues about it? because the button component already removes the outline.

https://github.com/bigbluebutton/bigbluebutton/blob/v2.0.x-release/bigbluebutton-html5/imports/ui/components/button/styles.scss#L84

Reply:
Yes, without this the outline is displayed on button focus. I did it based on the audio button. (https://github.com/bigbluebutton/bigbluebutton/blob/v2.0.x-release/bigbluebutton-html5/imports/ui/components/audio/audio-controls/styles.scss#L25)